homeassistant/components/reolink/__init__.py

```diff
@@ -79,6 +79,19 @@ async def async_device_config_update():
                     f"Error updating Reolink {host.api.nvr_name}"
                 ) from err
 
+        async with async_timeout.timeout(host.api.timeout):
+            try:
+                await host.renew()
+                host.lost_subscription = False
+            except ReolinkWebhookException as err:
```

Comment:
Since we won't raise `UpdateFailed` here, I think it may make more sense to handle the exception and conditional logging in the `ReolinkHost` class. Then we don't need to modify an attribute of the `ReolinkHost` instance from outside.